### DISCUSSION TITLE
How should I proceed to directly reference a CSS file and use it as a string in rsbuild?

### DISCUSSION THREAD

**Discussion by xieccc** [web-infra-dev/rsbuild#3749] [Q&A] [answered]
[Here's my code](https://codesandbox.io/p/devbox/3rh4vh?file=%2Fsrc%2FApp.tsx)

In Vite, I can append `?inline` to the CSS URL to import CSS. But how should I do it in rsbuild?

**Comment by SoonIter** [comment, marked as answer]
for `?raw` -> https://github.com/web-infra-dev/rsbuild/issues/3435  workaround: `import str from "!./myType.ts?raw";`

As for `?inline`, one workaround is to use [inline-loader](https://webpack.js.org/concepts/loaders/#inline) or set css-loader [export type](https://github.com/webpack-contrib/css-loader#string-3)

**Reply by SoonIter** [reply]
We are planning to support it through `import assert`(aka. `import with`) soon

**Reply by SoonIter** [reply]
🫡 https://github.com/web-infra-dev/rsbuild/issues/3778
